src/res/flame.html

```diff
@@ -262,26 +556,74 @@ <h1>/*title:*/</h1>
 		search(false);
 	}
 
+	btnInvert.onclick = function(e) {
+		invertMode();
+		e.preventDefault();
+	}
+
+	btnInvert.dblclick = function(e) {
+		e.preventDefault();
+	}
+
+	btnInvert.onmousedown = function(e) {
+		e.preventDefault();
+		e.stopPropagation();
+	}
+
+	document.onclick = function () {
+		ctxMenu.style.display = 'none';
+	}
+
+	ctxResetHighlight.onclick = function(e) {
+		e.stopPropagation();
+		focusOnType = null;
+		render(root);
+		ctxMenu.style.display = 'none';
+	}
+
 	window.onkeydown = function(event) {
-		if ((event.ctrlKey || event.metaKey) && event.key === 'f') {
-			event.preventDefault();
-			search(true);
-		} else if (event.key === 'Escape') {
+		if (event.key === 'Escape') {
 			search(false);
-		} else if ((event.key === 'n' || event.key === 'N') && nav.length > 0) {
-			navIndex = (navIndex + (event.shiftKey ? nav.length - 1 : 1)) % nav.length;
-			render(nav[navIndex]);
-			document.getElementById('matchval').textContent = matchval + ' (' + (navIndex + 1) + ' of ' + nav.length + ')';
-			window.scroll(0, inverted ? root.level * 16 : canvasHeight - (root.level + 1) * 16);
-			canvas.onmousemove();
+		} else if(event.ctrlKey || event.metaKey) {
+			switch(event.key) {
```

Comment:
Rewriting the function with two `switch` statements did not make it any cleaner or more readable. On the contrary, this excessively increased nesting level. IMO, switches do not make sense here, especially with one label only.

Reply:
There used to be more cases. switch makes it clearer as opposed to composite if-conditions. Converting to `if` chain.